rabi_splitter_WPF/MainWindow.xaml.cs

```diff
@@ -83,6 +83,34 @@ private void ReadMemory()
 
                 #endregion
 
+                #region Detect Reload
+
+                bool reloaded = false;
+                {
+                    int playtime = MemoryHelper.GetMemoryValue<int>(process, StaticData.PlaytimeAddr[mainContext.veridx]);
+                    reloaded = playtime != 0 && playtime < mainContext.lastplaytime;
+                    if (reloaded) DebugLog("Reload Game!");
+                    mainContext.lastplaytime = playtime;
+                }
+
+                #endregion
+
+                #region Detect Start Game
+
+                {
+                    int blackness = MemoryHelper.GetMemoryValue<int>(process, StaticData.BlacknessAddr[mainContext.veridx]);
+                    if (mainContext.previousBlackness == 0 && blackness >= 100000)
```

Comment:
the blackness value is count down from 100000 iirc, if we miss it, what will happen?